### PR TITLE
Added command for showing dog and cat photo

### PR DESCRIPTION
**Pull request by kimwonjin97**
Added code for showing randomly generated dog and cat photos in web-browsers

### CODE REVIEW

File: alexis.py [hunk 1]
```diff
@@ -252,6 +252,24 @@ def upperToCapitalize(txtAct):
                 except:
                     print(random.choice(resconst.errorResponse))
 
+            # Show 
+            elif "show" in command:
+                try:
+                    obj = command[5:]
```
Comment (alexismurari) [inline alexis.py:258]: maybe show a list of all the options if user only types "show"

**Review by shreyar7 — APPROVED** [review]
Looks great!

**Comment by alexismurari** [comment]
Looks good to me! 🔥  
Just an extra feature we could add if you want I mentioned it as comment here:
https://github.com/shreyar7/Alexis/pull/1#discussion_r768285734

File: alexis.py (hunk 1 above)
Comment (kimwonjin97) [reply alexis.py:258]: sounds good

**Pull request merged**